tests/performance-tests/src/tests/public-api/data-set-query/config.ts

```diff
@@ -0,0 +1,108 @@
+import { Options } from 'k6/options';
+import { QueryGeneratorConfig } from './queryGenerators';
+import {
+  dataSetQueryComparableOperators,
+  dataSetQueryIdOperators,
+} from '../../../utils/publicApiService';
+import steadyRequestRateProfile from '../../../configuration/steadyRequestRateProfile';
+import spikeProfile from '../../../configuration/spikeProfile';
+import rampingRequestRateProfile from '../../../configuration/rampingRequestRateProfile';
+import sequentialRequestProfile from '../../../configuration/sequentialRequestsProfile';
+import { parseIntOptional } from '../../../utils/utils';
+
+const profile = (__ENV.PROFILE ?? 'sequential') as
+  | 'load'
+  | 'spike'
+  | 'stress'
+  | 'sequential';
+
+const queries = (__ENV.QUERIES ?? 'simple') as 'simple' | 'complex';
+
+const limitToDataSetNames = __ENV.DATA_SET_NAMES?.split(',');
+const maxDataSetRows = parseIntOptional(__ENV.DATA_SET_MAX_ROWS) || undefined;
+const maxResultsToReturnPerDataSet =
+  parseIntOptional(__ENV.MAX_RESULTS_PER_DATA_SET) || 10000;
+
+function getOptions(): Options {
+  switch (profile) {
+    case 'load': {
+      return steadyRequestRateProfile({
+        mainStageDurationMinutes: 10,
+        cooldownStageDurationMinutes: 10,
+        steadyRequestRatePerSecond: 10,
+      });
+    }
+    case 'spike': {
+      return spikeProfile({
+        preSpikeStageDurationMinutes: 1,
+        spikeStageDurationMinutes: 1,
+        postSpikeStageDurationMinutes: 8,
+        normalTrafficRequestRatePerSecond: 10,
+        spikeRequestRatePerSecond: 30,
+      });
+    }
+    case 'stress': {
+      return rampingRequestRateProfile({
+        mainStageDurationMinutes: 10,
+        cooldownStageDurationMinutes: 10,
+        maxRequestRatePerSecond: 40,
+      });
+    }
+    case 'sequential': {
+      return sequentialRequestProfile({
+        mainStageDurationMinutes: 10,
+      });
+    }
+    default: {
+      throw Error(`Unknown profile '${profile}'`);
```

Comment:
Don't need to specify curly brackets in this case since the body isn't going onto multiple lines I believe (judging by the linting rules)

Reply:
I will check that out - thanks.

Reply:
Done ta - linter happy with this as you suspected.